docs/usage/quickstart.md

```diff
@@ -1,50 +1,51 @@
 (using_from_javascript)=
 
-# Using Pyodide from Javascript
-
-This document describes using Pyodide directly from Javascript.
+# Getting started using Pyodide
 
 ## Startup
 
-To include Pyodide in your project you can use the following CDN URL,
-
+To include Pyodide in your project you can use the following CDN URL:
+```{eval-rst}
   https://cdn.jsdelivr.net/pyodide/v0.17.0a2/full/pyodide.js
+```
 
-You can also download a release from
-[Github releases](https://github.com/pyodide/pyodide/releases)
-(or build it yourself), include its contents in your distribution, and import
-the `pyodide.js` file there from a `<script>` tag. See the following section on
-{ref}`serving_pyodide_packages` for more details.
+You can also download a release from [Github
+releases](https://github.com/pyodide/pyodide/releases) or build Pyodide
+yourself. See {ref}`serving_pyodide_packages` for more details.
 
-The `pyodide.js` file has a single `Promise` object which bootstraps the Python
-environment: `languagePluginLoader`. Since this must happen asynchronously, it
-is a `Promise`, which you must call `then` on to complete initialization. When
-the promise resolves, Pyodide will have installed a namespace in the global
-scope called {js:mod}`pyodide`.
+The `pyodide.js` file defines a single async function called
+{any}`loadPyodide <globalThis.loadPyodide>` which sets up the Python
+environment. When the `loadPyodide` function finishes, Pyodide installs global
+namespace called {js:mod}`pyodide`.
 
 ```pyodide
-languagePluginLoader.then(() => {
+async function main() {
+  await loadPyodide({ packageIndexURL : "https://cdn.jsdelivr.net/pyodide/v0.17.0a2/full/" });
```

Comment:
Maybe we could ask for another opinion on this parameter name from someone?  We won't be able to change it later. I know I suggested it, but I can't say I'm particularly happy with the current name either. Maybe I should look what names similar parameters have in other libraries.

Reply:
`packageIndexURL` is at least better than `baseURL`. We can change it later by adding a new named option which is synonymous to the current one and changing the docs to reflect the new name while keeping the old option as a synonym for backwards compatibility. No doubt that it's preferable to pick a good name the first time though.

Reply:
So,
 - pip has a `index-url` setting https://pip.pypa.io/en/stable/user_guide/#config-file
 - conda uses the term "channel_url" https://conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#show-channel-urls-show-channel-urls
 - nmp uses the "registry url" term https://docs.npmjs.com/configuring-your-registry-settings-as-an-npm-enterprise-user#setting-your-companys-npm-enterprise-registry-as-your-default-registry
 - cargo uses both registry and index terms https://doc.rust-lang.org/cargo/reference/registries.html#using-an-alternate-registry

Overall how about going with a `indexURL` parameter? Which is more or less what pip and cargo have. It's a bit shorter/ more readable than `packageIndexURL`.

Reply:
I'm happy with `indexURL`.

Reply:
OK, I renamed the parameter as I would like to merge this PR (since I have another one that depends on it)